doc/collections/_pages/content/handbook/setup--windows/chapter-1/Readme.md

```diff
@@ -0,0 +1,283 @@
+
+&nbsp;&nbsp;
+
+// NOTE:
+
+&nbsp;&nbsp;
+
+> Currently, the Windows server SKUs don’t ship with the Microsoft Store client enabled. This creates challenges for updating applications delivered by the Microsoft Store. As the Windows Package Manager ships as a part of the Desktop App Installer, and that is also not generally default on server SKUs [&hellip;]
+
+&nbsp;&nbsp;
+
+A version with the Winget tool winds up as a release and MSIX package on GitHub (since version 1.x). There exists the latest stable and pre-releases:
+
+&nbsp;&nbsp;
+
+| Assets                                                 |
+| ------------------------------------------------------ |
+| 7bcb1a0ab33340daa57fa5b81faec616_License1.xml          |
+| DesktopAppInstallerPolicies.zip                        |
+| Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle |
+| Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.txt        |
+
+&nbsp;&nbsp;
+
+Refs: https://github.com/microsoft/winget-cli/releases
+
+&nbsp;&nbsp;
+
+If you download the MSIXbundle file, you will find that you can’t simply double-click the file and have it install in Windows Server 2022, as one would in Windows 10, due to no store availability in Windows Server.
+
+&nbsp;&nbsp;
+
+The reason for this? Microsoft claims that they do not support it, so all of which follows should be considered as experimental
+
+&nbsp;&nbsp;
+
+However, we can use the `Add-AppxPackage` PowerShell cmdlet to install the MSIXbundle file.
+
+&nbsp;&nbsp;
+
+When attempting to install MSIXbundle file using this method, there is a slight hurdle to overcome
+as there will be errors as shown below.
+
+&nbsp;&nbsp;
+
+> Windows cannot install package Microsoft.DesktopAppInstaller_1.19.3132.0_x64\_\_8wekyb3d8bbwe because this package
+> depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by
+> "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor
+> architecture and minimum version 7.2109.13004.0, along with this package to install. The frameworks with name
+> "Microsoft.UI.Xaml.2.7" currently installed are: {}
+
+&nbsp;&nbsp;
+
+[^1]: https://learn.microsoft.com/en-us/windows/package-manager/winget/
+
+&nbsp;&nbsp;
+
+`MakeAppx.exe` is a tool available in the Windows 10 SDK that allows for packaging and bundling of MSIX packages.
+
+&nbsp;&nbsp;
+
+`MakeAppx.exe` can be used to extract the file contents of a Windows 10 app package or bundle. It also encrypts and decrypts app packages and bundles.
+
+&nbsp;&nbsp;
+
+After the Windows 10 SDK is installed, depending on where it was installed,
+`MakeAppx.exe` is usually found here:
+
+&nbsp;&nbsp;
+
+`C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\x64\makeappx.exe`
+
+&nbsp;&nbsp;
+
+Add this directory to your `PATH`
```

Comment:
This is not as straightforward as it may sound, but would like to include something in PowerShell